Makefile.config.in

```diff
@@ -260,11 +260,12 @@ endif # ifeq "$(TOOLCHAIN)" "msvc"
 # User code should absolutely not rely on their presence because
 # in the future their definition may be moved to a more private part of
 # the compiler's build system
+DIFF = diff
```

Comment:
This should move away from the comment above it up to around L242 (below `NAKED_POINTERS`)